docs/topic/security.rst

```diff
@@ -38,16 +38,22 @@ command on the terminal. No password required.
 
 This is a **lot** of power, and they can do pretty much anything they want to
 the server - look at other people's work, modify it, break the server in cool &
-funky ways, etc. This also means if an admin's credentials are compromised (
-easy to guess password, password re-use, etc) the entire JupyterHub is compromised.
+funky ways, etc. This also means **if an admin's credentials are compromised (
+easy to guess password, password re-use, etc) the entire JupyterHub is compromised.**
 
 Off-boarding users securely
 ===========================
 
 When you delete users from the JupyterHub admin console, their unix user accounts
 are **not** removed. This means they might continue to have access to the server
 even after you remove them from JupyterHub. Admins should manually remove the user
-from the server & archive their home directories as needed. If the user removed
+from the server & archive their home directories as needed. For example, the
+following command deletes the user ``yuvi``.
+
+.. code-block::
```

Comment:
I think this should be:

sudo userdel jupyter-yuvipanda

to delete a user named 'yuvipanda' from the jupyterhub.

Reply:
ooops! good point